core/src/main/java/org/kiwix/kiwixmobile/core/main/WebViewCallback.java

```diff
@@ -35,4 +36,6 @@
   void webViewPageChanged(int page, int maxPages);
 
   void webViewLongClick(String url);
+
+  void onToggledFullscreen(@NotNull Boolean isFullScreen);
```

Comment:
Lets call this something more clear because it is conflicting with the existing feature of fullscreen webview.
`onFullscreenVideoToggled` and get this boxed `Boolean` out of here, the primitive is better and doesn't need the not null annotation because it can't be null